Tests/Marketplace/marketplace_services.py

```diff
@@ -842,10 +844,78 @@ def upload_to_storage(self, zip_pack_path, latest_version, storage_bucket, overr
             print_error(f"Failed in uploading {self._pack_name} pack to gcs. Additional info:\n {e}")
             return task_status, True, None
 
-    def prepare_release_notes(self, index_folder_path, build_number):
+    def get_changelog_latest_rn(self, changelog_index_path: str) -> Tuple[dict, LooseVersion]:
         """
-        Handles the creation and update of the changelog.json files.
+        Returns the changelog file contents and the last version of rn in the changelog file
+        Args:
+            changelog_index_path (str): the changelog.json file path in the index
+
+        Returns: the changelog file contents and the last version of rn in the changelog file
+
+        """
+        print_color(f"Found Changelog for: {self._pack_name}", LOG_COLORS.NATIVE)
+        with open(changelog_index_path, "r") as changelog_file:
+            changelog = json.load(changelog_file)
+
+        # get the latest rn version in the changelog.json file
+        changelog_rn_versions = [LooseVersion(ver) for ver in changelog]
+        changelog_rn_versions.sort()
+        # no need to check if changelog_rn_versions isn't empty because changelog file exists
+        changelog_latest_rn_version = changelog_rn_versions[-1]
```

Comment:
Not critical but still, no need to sort in order to get the max value
```suggestion
        changelog_latest_rn_version = max(changelog_rn_versions)
```